server/main.go

```diff
@@ -424,3 +426,13 @@ func getBlock(w http.ResponseWriter, r *http.Request) {
 	}
 	writeJSON(w, http.StatusOK, block)
 }
+
+func pingDB(w http.ResponseWriter, r *http.Request) {
+	err := backendInstance.PingDB()
+	if err != nil {
+		errorResponse(w, http.StatusInternalServerError, err)
+	} else {
+		writeJSON(w, http.StatusOK, "PingOK")
```

Comment:
Looks like there shouldn't be a body: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/HEAD
Technically should be headers from GET request, but we only need the 200 code for our purposes.